README.md

```diff
@@ -6,6 +6,10 @@
 >
 >Web Dev portfolio to showcase coding skills and experience building websites.
 
+## Link to My Portfolio
+
+[Go to My Portfolio](https://trictonicmp.github.io/Portfolio/)
```

Comment:
- [ ] [Optional] It would be more professional if you add your deployment link under `Live Demo` title not `Link to My Portfolio` like this:

`## Live demo`
   `My portfolio live demo [link](https://trictonicmp.github.io/Portfolio/).`


and it will look like that

## Live demo
   My portfolio live demo [link](https://trictonicmp.github.io/Portfolio/).